app/views/system_status/index.html.haml

```diff
@@ -0,0 +1,66 @@
+.container.queues
+  .module
+    .section-header
+      %h3 Queues Overview
```

Comment:
Use i18n for all the text.

Reply:
Done! Any other changes?

Reply:
Not at the moment! Is this a complete MVP? If so, mark it ready for review and I can probably review it tomorrow. Also @Formasitchijoh please do a code review if you have a chance.